index.js

```diff
@@ -93,7 +93,7 @@ module.exports = fp(
         if (raw.socket.destroyed) {
           throw new Errors.SOCKET_CLOSED(reqId)
         } else {
-          raw.once(
+          raw.socket.once(
             'close',
             function () {
               if (controllers.has(this)) {
```

Comment:
I tried checking if the request was completed by doing
```suggestion
              if (!raw.completed && controllers.has(this)) {
```
but that doesn't seem to work as intended for a http1.1 client like `unidici`. 

Do we need some other check, as i'm not sure if it will work for http2 clients, as they can close their sending side of the socket while keeping the receiving side open?